session106/XMap/XMap/Model/Parsing/NearbyStationParsing.swift

```diff
@@ -8,11 +8,15 @@
 
 import Foundation
 
-struct NearbyStation: Codable {
+struct Station: Codable {
     
-    let commonName: String
-    let distance: Double
-    let lat: Double
-    let long: Double
-    let stationNaptan: String
+    struct StopPoint: Codable {
+        let commonName: String
+        let distance: Double
+        let lat: Double
+        let lon: Double
+        let stationNaptan: String
+    }
+    
+    let stopPoints : [StopPoint]
```

Comment:
`StopPoint` is a `Station`, maybe just remove StopPoints and have a simple station, so we can have an array of stations. At parsing, you can just get "stopPoints" as an Array<Any>, similarly what you did for tube.